### PR TITLE
Enhancement #7208 done

### PR DESCRIPTION
**Pull request by amit-webkul**
#7208 
> Add arrow class for status filter.

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
Please **compile** the files, Please **run** the **below command** before pushing your code.

**_npm run prod_**

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@amitwebkul 
The icon should be **slightly bold.** please find the **screenshot** [Here](https://prnt.sc/Cj4nSFtByFx2)

**Comment by Sanjaybhattwebkul** [comment]
@amitwebkul 
This is **working fine,** Please find the **screenshot** [Here](https://prnt.sc/eGtj-Q-f_vSC)

File: packages/Webkul/Ui/reflect [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Module not found: Error: Can't resolve 'on' in '/home/users/amit.kumar/www/html/amit-bagisto/packages/Webkul/Ui'
+ @ multi - the changes on ui main[3]
+
+ERROR in multi - the changes on ui
+Module not found: Error: Can't resolve 'the' in '/home/users/amit.kumar/www/html/amit-bagisto/packages/Webkul/Ui'
+ @ multi - the changes on ui main[1]
+
+ERROR in multi - the changes on ui
+Module not found: Error: Can't resolve 'ui' in '/home/users/amit.kumar/www/html/amit-bagisto/packages/Webkul/Ui'
+ @ multi - the changes on ui main[4]
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/reflect:37]: Why this file is pushed?

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/filters/column-filter.vue [hunk 2]
```diff
@@ -262,13 +262,16 @@
                             class="dropdown-btn"
                             @click="toggleCheckboxDropdown"
                         >
+                        <span class="button-span">
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/filters/column-filter.vue:265]: Add new line.

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/filters/column-filter.vue [hunk 3]
```diff
@@ -262,13 +262,16 @@
...
                             {{
                                 types.checkbox.value.length > 0
                                     ? types.checkbox.value.join(', ')
                                     : translations.select
                             }}
-
-                            <i class="icon arrow-down-icon"></i>
+                        </span>
+                        <span>
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/filters/column-filter.vue:272]: Same here.

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/app.scss [hunk 4]
```diff
@@ -2768,3 +2768,19 @@
 .nested {
     display: none;
 }
+
+.dropdown-btn .button-span {
+    display: inline-flex;
+    width: 92%;
+}
+.drop-down-arrow {
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/app.scss:2776]: New line.

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/app.scss [hunk 5]
```diff
@@ -2768,3 +2768,19 @@
...
+    border:2px solid rgb(40, 40, 40);
+    border-width: 0 3px 3px 0 !important;
+    padding: 2.6px;
+    transform: rotate(45deg);
+}
+@media only screen and (max-width: 1440px) {
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/app.scss:2782]: New line.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by jitendra-webkul** [comment]
Resolve conflicts

**Comment by Sanjaybhattwebkul** [comment]
@amit-webkul 
Resolve the **conflicts**.

**Comment by Sanjaybhattwebkul** [comment]
@amit-webkul 
Resolve the **conflicts**

**Comment by Sanjaybhattwebkul** [comment]
@amit-webkul 
Getting **duplicate commits.** please **resolved**

**Comment by Sanjaybhattwebkul** [comment]
**1.** Git fetch --all
**2.** git reset --hard 38f7cd3
**3.** git push -u origin filter-ui-issue -f

**Pull request closed without merging**